aten/src/ATen/native/cuda/PointwiseOpsKernel.cu

```diff
@@ -30,17 +30,17 @@ void addcdiv_cuda_kernel(TensorIterator& iter, Scalar value) {
   });
 }
 
-void smooth_l1_backward_cuda_kernel(TensorIterator& iter, Scalar norm) {
+void smooth_l1_backward_cuda_kernel(TensorIterator& iter, Scalar norm, beta beta) {
```

Comment:
this looks like a typo.  Also the same double issue.

Reply:
I finally got cuda to build successfully this morning and point out this error to me :)